docs/documentation/patterns/empty-states.mdx

```diff
@@ -0,0 +1,217 @@
+      />
+    </Pane>
+  }
+>
+  <Button>Show Empty State</Button>
+</Popover>
+```
+
+### Minimal
+
+The smallest acceptable empty state uses only a visual and headline. Minimal empty states typically occur in small components, like drop downs or side sheets within tables.
+
+```jsx
+<Popover
+  content={
+    <Pane width={300} height="auto">
+      <EmptyState
+        background="light"
+        title="No event selected"
+        orientation="vertical"
+        icon={<HandUpIcon color="#C1C4D6" />}
+        iconBgColor="#EDEFF5"
+      />
+    </Pane>
+  }
+>
+  <Button>Show Empty State</Button>
+</Popover>
+```
+
+### Calls to Action
+
+<RuleLayout>
+  <RuleCard
+    type="Do"
+    description="Use a primary button if the action allows the user to resolve the empty state."
+    image="/empty-state-cta-do.png"
+  />
+  <RuleCard
+    type="Don't"
+    description="Offer a CTA twice on the same screen. Hide the top line primary button when the action is redundant."
+    image="/empty-state-cta-dont.png"
+  />
+</RuleLayout>
+<RuleLayout>
+  <RuleCard
+    type="Do"
+    description="Consider a text link to provide additional information if a button cannot resolve the empty state."
+    image="/empty-state-textlink-do.png"
+  />
+</RuleLayout>
+<br />
+<br />
+<br />
+
+### Content
+
+Empty state content should explain what information or data isn’t being showed, and how to change the empty state to a populated state.
+
+#### Headline text
+
+Limit headline text to one line. Since users typically skim online content, the headline should contain the most important information.
+
+- For an onboarding experience: Explain what eventually populates the empty state.
+- For an error experience: Explain what went wrong.
+- For a no results/data experience: Explain what happened.
+  <br />
+  <br />
+
+#### Body copy (optional)
+
+Limit body copy to two lines. Use the body copy to introduce additional information about what the user can do with this UI element.
+
+- For an onboarding empty state: Describe how a feature works or what will happen when they use it.
+- For an error empty state: Explain how the user can fix the error.
+- For a no results/data experience: Explain what they can do to try and get a different outcome.
+  <br />
+  <br />
+
+#### Buttons and links (optional)
+
+Buttons and link text should set expectations for the user.
+
+- Button text should reflect what will happen when the user selects the button.
+- Link text should reflect what kind of info the user gets at the destination.
+  <br />
+  <br />
+
+#### Tone
+
+Tone should be educational and plain-spoken.
+
+<RuleLayout>
+  <RuleCard type="Do" description="Use an educational and plain-spoken tone." image="/empty-state-tone-do.png" />
+  <RuleCard
+    type="Don't"
+    description="Use excessive emotion or exceed 1 line for headline and 2 lines for body."
+    image="/empty-state-tone-dont.png"
```

Comment:
One thing that sticks out to me - would it be worth actually having some JSX in here instead of a static image?

Reply:
Like SVG in JSX? We can certainly do that, just need more prep from the design side. Is the main benefit scalability?

Reply:
Gonna stick with static image for now until we see further use cases.